packages/core/data-transfer/lib/providers/local-file-destination-provider.ts

```diff
@@ -94,25 +95,12 @@ class LocalFileDestinationProvider implements IDestinationProvider {
     }
 
     // FS write stream
-    streams.push(createMultiFilesWriteStream(filePathFactory, this.options.file.maxSize));
+    streams.push(createMultiFilesWriteStream(filePathFactory, this.options.file.maxSizeJsonl));
```

Comment:
We'll merge https://github.com/strapi/strapi/pull/14766 first and then I'll resolve the conflicts here.